File: Assignment No_36/Question_1.java
/* 1.Write a Java program which accept string from user and count number of capital characters.

Input : “Marvellous Multi OS”
Output : 4

Input : "Gawali Ganesh"
Output : 2

*/
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////


import java.util.*;

class Ass36_1
{
    public static void main(String Arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter The String : ");
        String name = sobj.nextLine();

        int i = 0, iCnt = 0;
        char ch;

        for(i = 0; i < name.length(); i++)
        {
            ch = name.charAt(i);

            if((ch >= 'A') && (ch <= 'Z'))
            {
                iCnt++;
            }
        }

        System.out.println("Capital characters are : "+iCnt);
    }
}

File: Assignment No_27/Question_1.c
/*1.Write a program which accept string from user and count number of capital characters.

Input : "Marvellous Multi OS"
Output : 4

*/
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<stdio.h>

int  CountCapital(char *str)
{
	int iCnt=0;

	while(*str !='\0')
	{
	     if(*str>='A' && *str<='Z')
	     iCnt++;

	     str++;
	}
	return iCnt;
}

int main()
{
   char Arr[20];
   int iRet=0;

   printf("Enter the string:\n");
   scanf("%[^'\n']s",Arr);

   iRet=CountCapital(Arr);

   printf("Capital characters are:%d\t",iRet);

   return 0;
}
